### PR TITLE
Poker Restyle and Responsiveness

### PR DESCRIPTION
**Pull request by chankk**
* Expanded poker to full width
* Made poker _more_ responsive
* Hide chat and the cheatsheet on smaller screens
* Restructured poker components to use a Bootstrap's grid (row and column) layout
* Replaced the Raise input

### CODE REVIEW

**Comment by rzayevan** [comment]
Is Devon working on this branch as well? Or how are you guys sharing the work?

**Review by wpasadilla — APPROVED** [review]
Good. Would like to approve for testing.

File: poker-pals/src/components/pokerComponents/RaiseToggle.vue [hunk 1]
```diff
@@ -1,140 +0,0 @@
-<template>
-    <div class="raiseToggle"> <!--the container holding the raise input, it contains two buttons, a slider, and a value display-->
```
Comment (rzayevan) [inline poker-pals/src/components/pokerComponents/RaiseToggle.vue:2]: Is this file not needed anymore?

**Review by rzayevan — APPROVED** [review]

**Review by darbrius — APPROVED** [review]

File: poker-pals/src/components/pokerComponents/RaiseToggle.vue (hunk 1 above)
Comment (chankk) [reply poker-pals/src/components/pokerComponents/RaiseToggle.vue:2]: nope! replaced it with a number input (:

**Pull request merged**